inference_perf/config.py

```diff
@@ -59,6 +59,8 @@ class SharedPrefix(BaseModel):
     system_prompt_len: int = 100
     question_len: int = 50
     output_len: int = 50
+    # create user session for each group. The chat context will be appended for the each request in the group.
```

Comment:
Would be good to update the `config.md` and the main README to mention this is a supported feature.